integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshService.java

```diff
@@ -2,69 +2,62 @@
-import net.schmizz.sshj.transport.TransportException;
 import net.schmizz.sshj.transport.verification.PromiscuousVerifier;
-import net.schmizz.sshj.userauth.UserAuthException;
 
+@Component
 public class SshService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SshService.class);
 
-    public int countFilesOnHostByExtensionAndPath(@Nonnull List<String> publicIps, @Nonnull String[] paths, Optional<String> defaultCommandChain,
-                    String privateKey, String extension, String sshUser, int sshTimeout, int require) throws IOException {
-        int quantity = 0;
-        for (String publicIp : publicIps) {
-            for (String path : paths) {
-                String fileListCommand = String.format("find %s -type f -name *.%s", path, extension);
-                String formatTextForFullCommand = !defaultCommandChain.isPresent() || defaultCommandChain.get().endsWith(";") ? "%s%s" : "%s;%s";
-                String command = String.format(formatTextForFullCommand, defaultCommandChain.isPresent() ? defaultCommandChain : "", fileListCommand);
-                try (SSHClient sshClient = createSshClient(publicIp, sshTimeout, sshUser, privateKey)) {
-                    Pair<Integer, String> cmdOut = execute(sshClient, command);
-                    quantity += Stream.of(cmdOut.getValue().split(System.getProperty("line.separator"))).filter(s -> s != null && s.startsWith("/")).count();
-                } catch (UserAuthException | TransportException ignore) {
-                    LOGGER.warn("Whoops! SSH fail on {} while getting info for location: {}", publicIp, path);
-                }
-            }
-        }
-        return quantity;
+    @Inject
+    private CommonCloudProperties commonCloudProperties;
+
+    public String getRecipeExecutionOutput(String publicIp, String sshUser, String filePath) {
+        SSHClient sshClient = createSshClient(publicIp, sshUser);
+        return executeCommand(sshClient, "cat " + filePath);
     }
 
-    private static SSHClient createSshClient(String host, int sshTimeout, String sshUser, String defaultPrivateKeyFile) throws IOException {
-        SSHClient client = new SSHClient();
-        client.addHostKeyVerifier(new PromiscuousVerifier());
-        client.connect(host, 22);
-        client.setConnectTimeout(sshTimeout);
-        client.authPublickey(sshUser, defaultPrivateKeyFile);
-        return client;
+    private SSHClient createSshClient(String host, String sshUser) {
+        try {
+            SSHClient client = new SSHClient();
+            client.addHostKeyVerifier(new PromiscuousVerifier());
+            client.connect(host, 22);
+            client.setConnectTimeout(commonCloudProperties.getSshTimeout());
+            client.authPublickey(sshUser, commonCloudProperties.getSshPrivateKeyPath());
+            return client;
+        } catch (IOException e) {
+            throw new IllegalStateException(e);
```

Comment:
I would log here something about what is going on...

Reply:
Not needed to be logged. Exception is logged by default.